src/Compilers/CSharp/Portable/Parser/LanguageParser_Patterns.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
+    using System;
     using Microsoft.CodeAnalysis.Syntax.InternalSyntax;
```

Comment:
Please move `using` statements to outer scope.

Reply:
Deleting `using System;` as it isn't needed. The other one needs to be nested to resolve a conflict with names in the current namespace. It is the same in `LanguageParser.cs`.